package.json

```diff
@@ -38,7 +38,7 @@
     "build": "web-ext build -s ./extension/ --overwrite-dest -n searchengine-devtools.xpi",
     "postbuild": "rm validate.js",
     "clean": "rimraf web-ext-artifacts extension/content/diff.js extension/content/validate.js",
-    "start": "npm run clean && npm run build && web-ext run -s extension"
+    "start": "npm run clean && npm run build && web-ext run -s extension --pref \"xpinstall.signatures.required=false\" --pref \"extensions.experiments.enabled=true\""
```

Comment:
I was just testing this and noticed there was a complaint on the console that `xpinstall.signatures.required` is not allowed to be set. It doesn't break it, but it looks like [it isn't necessary as it gets set automatically](https://github.com/mozilla/web-ext/blob/f814a6195ad8a40d3bca1fc8105473a94e79bdca/src/firefox/preferences.js#L53-L54)

Reply:
Thank you for noticing the warning. 
I've removed `xpinstall.signatures.required`.